### ISSUE TITLE
Cannot check checkboxes with spacebar

### ISSUE THREAD

**Issue by vierkantemeter** [rstaib/jquery-steps#101]
I'm building a form/wizard that has to be keyboard accessible. This is working great with Jquery Steps so far. The only problem I'm facing now is that checkboxes cannot be checked with the spacebar anymore. I tested the form without Jquery Steps and found that without this plugin I can check the checkboxes with the spacebar. 

What could be causing this? Thanks for any help!

**Comment by contactmatts** [comment]
To solve this issue, I found that setting suppressPaginationOnFocus to false upon step initialization fixed the issue.
